server/embedded/src/main/resources/log4j2.xml

```diff
@@ -301,7 +301,7 @@ account managers so that we can coordinate edits with other customized copies of
             <AppenderRef ref="QUERY_STATS"/>
         </Logger>
 
-        <Logger name="org.labkey.audit.event" level="OFF">
+        <Logger name="org.labkey.audit.event" additivity="false" level="OFF">
```

Comment:
I'm not super familiar with this logger, but clearly the intent is to log to both LABKEY_AUDIT and CONSOLE. I would remove both `additivity` and the CONSOLE ref, which should be the equivalent.